subgraph-js/src/schema-helper/buildSchemaFromSDL.ts

```diff
@@ -295,15 +299,24 @@ export function buildSchemaFromSDL(
     };
   }
 
+  const description = schemaDescriptions.length === 0 ? undefined :
```

Comment:
Similar comment as above - there should only be one description so I don't think we need to join anything.

Reply:
> Similar comment as above - there should only be one description so I don't think we need to join anything.

Agreed! I'll wait for a 🟢  light on the similar question above before proceeding with the suggested improvement 👍